app/views/sources/show.html.erb

```diff
@@ -8,13 +8,13 @@
                                           trailing_slash: true) %>' />
   <link rel="stylesheet" type="text/css" href= 'http://cdn.jsdelivr.net/jquery.slick/1.5.9/slick.css' />
   <link rel="stylesheet" type="text/css" href= 'http://cdn.jsdelivr.net/jquery.slick/1.5.9/slick-theme.css' />
+  <script type="text/javascript"
+          src="http://cdn.jsdelivr.net/jquery.slick/1.5.9/slick.min.js"
+          defer></script>
+  <%= javascript_include_tag 'carousel', defer: 'defer' %>
 <% end %>
 
 <% content_for :foot_script do %>
-  <%= javascript_include_tag 'lightbox' %>
```

Comment:
I deleted lightbox entirely because I don't think we are using it now that open seadragon has been implemented.  Is this correct?